Source/Tools/CommonTools/HarnessHelpers.h

```diff
@@ -419,14 +419,8 @@ class HarnessCodeLoader final : public FEX::CodeLoader {
     return RIP;
   }
 
-  bool MapMemory() {
+  bool MapMemory(std::function<void*(uint64_t, size_t)> DoMMap) {
```

Comment:
This should probably taken by const reference, since it's only ever called, so the copy isn't really necessary